internal/handlers/tigris/msg_delete.go

```diff
@@ -254,17 +254,30 @@ func deleteDocuments(ctx context.Context, dbPool *tigrisdb.TigrisDB, qp *tigrisd
 		ids[i] = map[string]any{"_id": json.RawMessage(id)}
 	}
 
+	collection := tigrisdb.EncodeCollName(qp.Collection)
+
 	var f driver.Filter
 	switch len(ids) {
 	case 0:
 		f = driver.Filter(`{}`)
+
+		resp, err := dbPool.Driver.UseDatabase(qp.DB).Delete(ctx, collection, f)
+		if err != nil {
+			return 0, lazyerrors.Error(err)
+		}
+
+		if err := dbPool.Driver.UseDatabase(qp.DB).DropCollection(ctx, collection); err != nil {
```

Comment:
Is the dropCollection to reset the schema? Should we then just call Drop? I guess you need count that's why you're calling Delete first.

Reply:
Right, return count and reset the schema.